build.gradle.kts

```diff
@@ -0,0 +1,31 @@
+plugins {
+    kotlin("jvm") version "1.7.10"
+    `java-gradle-plugin`
```

Comment:
If it turns out that `ReflectProperties.java` is not needed (or can be easily rewritten in Kotlin), this won't be needed any longer, right?